shared/Gemfile

```diff
@@ -6,6 +6,6 @@ gem "gapic-generator", path: "../gapic-generator"
 gem "gapic-generator-cloud", path: "../gapic-generator-cloud"
 gem "googleapis-common-protos", [">= 1.3.9", "< 2.0"]
 gem "googleapis-common-protos-types", [">= 1.0.4", "< 2.0"]
-gem "google-gax", "~> 1.0"
+gem "google-gax", github: "blowmage/gax-ruby", branch: "2.0/refactor-api_call"
```

Comment:
Before this is merged I would like to change this to reference master. So the GAX PR should be merged first, and we should see one green build referencing master before merging.